apps/storefront/src/index.html

```diff
@@ -1,8 +1,6 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-  </head>
+  <meta charset="UTF-8" />
```

Comment:
We prepared for it. I fixed meta service, so it's work as with header as without

Reply:
i'm in favour of keeping the `head` element, to make it more obvious that we use standard html

Reply:
We had this template at start. Head was adding  by me in case of `page head service`, I thought it was a limitation so I fixed this moment.
https://github.com/spryker/oryx/commit/860ad6f524feb19b8776bb572827790de26d6f00#diff-c87a2a62fcdc8aa20f501092830edff7a5a5184162cfbb287717cfff2c464bed